docs/rst/knora-ontologies/knora-base.rst

```diff
@@ -0,0 +1,1544 @@
+.. Copyright © 2015 Lukas Rosenthaler, Benjamin Geer, Ivan Subotic,
+   Tobias Schweizer, André Kilchenmann, and André Fatton.
+
+   This file is part of Knora.
+
+   Knora is free software: you can redistribute it and/or modify
+   it under the terms of the GNU Affero General Public License as published
+   by the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   Knora is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU Affero General Public License for more details.
+
+   You should have received a copy of the GNU Affero General Public
+   License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
+
+.. _knora-base:
+
+***********************
+The Knora Base Ontology
+***********************
+
+.. contents:: :local:
+
+Introduction
+============
+
+Resource Description Framework (RDF)
+------------------------------------
+
+Knora uses a hierarchy of ontologies based on the Resource Description
```

Comment:
We used to explain what Knora stands for: Knowledge Organization, Representation, and Annotation, http://knora.org

But maybe since this is not a standalone document anymore, we can assume that the reader has learned about that in another section.